File: javaScript/programmers/level_1/k번째수.js
function solution(arr, commands) {  
    let newArr = [], answer = [];
    for(let i=0; i<commands.length; i++){
        newArr = arr.slice(commands[i][0]-1, commands[i][1]).sort((a,b) => a - b);
        answer.push(newArr[commands[i][2]-1]);
    }
    return answer;
}

File: python/programmers/level_1/K번째수.py
def solution(array, commands):
    tmpArr = []
    answer = []
    for i in commands:
        x = list(array[i[0]-1:i[1]])
        x.sort()
        tmpArr.append(x)

    for j in range(len(tmpArr)):
        answer.append(tmpArr[j][commands[j][2]-1])
    return answer
